test/integration/restrictusers_test.go

```diff
@@ -16,20 +16,12 @@ import (
 )
 
 func TestRestrictUsers(t *testing.T) {
-	masterConfig, err := testserver.DefaultMasterOptions()
+	masterConfig, clusterAdminKubeConfig, err := testserver.StartTestMaster()
```

Comment:
Since RBRs are now by default registered as an admission plugin, shouldn't `testutil.WaitForRoleBindingRestrictionCRDAvailable()` be called after kube-apiserver has started?

Reply:
I've added it.